src/js/video.js

```diff
@@ -553,6 +554,8 @@ videojs.computedStyle = computedStyle;
  */
 videojs.dom = Dom;
 
+videojs.domData = DomData;
```

Comment:
I am against this being in a production build of Video.js, though, I do see the usefulness of it when debugging.

Reply:
If we can have this be here during local dev but not when releasing, that's fine.

Reply:
For local development we can just import DomData (as long as we let it export `elData`). I think that plugins could also benefit from being able to see what `elData` they may be leaking, but I am not quite sure how to do that without exposing it here.

Reply:
Perhaps we can expose it somewhere else?

Reply:
I'm not sure we can safely expose it and not deal with consequences down the line or have issues with people fiddling with it and causing issues.